compliance/src/main/java/bio/terra/pearl/compliance/model/SingleUpdate.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.pearl.compliance.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@ToString
+public class SingleUpdate {
+
+    private String id;
+
+    private boolean inScope;
```

Comment:
One of the changes vanta made to the API was to allow you to set `inScope` for each `id` in the list.  Prior to this change, all resources in the payload had to have the same scope.  Although our code currently splits the list of resources into `inScope=true` and `inScope=false` lists, I decided to leave the splitting code alone since it's working just fine.